packages/components/src/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const SIZES = ['xs', 'sm', 'md', 'lg'] as const;
```

Comment:
please move this to the theme folder, if it only applies to the button you can just move it to the Button theme definition

Reply:
as I have seen it might work in different components so I moved it to the theme folder.